js/tests/index.test.js

```diff
@@ -0,0 +1,98 @@
+const TestWasm = require("./dist/test");
+
+const languages = [
```

Comment:
can we get this from lang.json (or whatever it is?) in semgrep-interfaces?

Reply:
We can, but since each language requires being wired up, I don't want people to have to update a bunch of semgrep js stuff when they add a new language. This way people can add a new language that's experimental w/out having to go through this whole slog